src/Collections/Sequences/Seq.dfy

```diff
@@ -266,8 +266,8 @@ module Seq {
   function method {:opaque} Remove<T>(s: seq<T>, pos: nat): (s': seq<T>)
     requires pos < |s|
     ensures |s'| == |s| - 1
-    ensures forall i {:trigger s'[i], s[i]} | 0 <= i < pos :: s'[i] == s[i]
-    ensures forall i {:trigger s'[i]} | pos <= i < |s| - 1 :: s'[i] == s[i+1]
+    ensures forall i | 0 <= i < pos :: s'[i] == s[i]
+    ensures forall i| pos <= i < |s| - 1 :: s'[i] == s[i+1]
```

Comment:
We lost one too many spaces at the start here.

Reply:
Fixed.